sdk/storage/Azure.Storage.Blobs/src/Models/BlobsModelFactory.cs

```diff
@@ -1561,40 +1561,40 @@ public static GetBlobTagResult GetBlobTagResult(Tags tags)
         /// Creates a new BlobDownloadDetails instance for mocking.
         /// </summary>
         public static BlobDownloadDetails BlobDownloadDetails(
-            BlobType blobType,
-            long contentLength,
-            string contentType,
-            byte[] contentHash,
-            DateTimeOffset lastModified,
-            IDictionary<string, string> metadata,
-            string contentRange,
-            string contentEncoding,
-            string cacheControl,
-            string contentDisposition,
-            string contentLanguage,
-            long blobSequenceNumber,
-            DateTimeOffset copyCompletedOn,
-            string copyStatusDescription,
-            string copyId,
-            string copyProgress,
-            Uri copySource,
-            CopyStatus copyStatus,
-            LeaseDurationType leaseDuration,
-            LeaseState leaseState,
-            LeaseStatus leaseStatus,
-            string acceptRanges,
-            int blobCommittedBlockCount,
-            bool isServerEncrypted,
-            string encryptionKeySha256,
-            string encryptionScope,
-            byte[] blobContentHash,
-            long tagCount,
-            string versionId,
-            bool isSealed,
-            IList<ObjectReplicationPolicy> objectReplicationSourceProperties,
-            string objectReplicationDestinationPolicy,
-            bool hasLegalHold,
-            DateTimeOffset createdOn)
+            BlobType blobType = default,
```

Comment:
We need to remote `= default` from all the parameters on the old overload.

Reply:
I assume you mean to say "remove `= default`"?
Gotcha makes sense. We can completely decommission the old factory method since the new one can replace it with the optional arguments